ui/apps/platform/src/Containers/Collections/CollectionsPage.tsx

```diff
@@ -26,10 +27,14 @@ function CollectionsPage() {
 
     if (validPageActionProp) {
         return (
-            <CollectionsFormPage
-                hasWriteAccessForCollections={hasWriteAccessForCollections}
-                pageAction={validPageActionProp}
-            />
+            <>
+                <PageSection className="pf-u-h-100" padding={{ default: 'noPadding' }}>
```

Comment:
Comparing to `CollectionsTablePage` below brings to mind a question: what is the reason to factor `PageSection` element out of `CollectionsFormPage`?

Reply:
Admittedly this is probably a detail lost in the shuffle. I agree it makes more sense to be in `CollectionFormPage` to match the usage of `CollectionsTablePage`.